apiserver/facades/agent/machine/machiner_test.go

```diff
@@ -315,17 +319,6 @@ func (s *machinerSuite) TestWatch(c *gc.C) {
 			{Error: apiservertesting.ErrUnauthorized},
 		},
 	})
-
-	// Verify the resource was registered and stop when done
```

Comment:
Note: removed the test of the actual watcher functionality, since this should be an API test (and there is no simple way to get the watcher out when its not linked in the resource).